src/main/java/games/strategy/engine/lobby/server/db/DbUserController.java

```diff
@@ -67,6 +80,12 @@ public void updateUser(final DBUser user, final HashedPassword password) {
     secondaryDao.updateUser(user, password);
   }
 
+  @Override
+  public void updateUser(final DBUser user, final String password) {
```

Comment:
Please get rid of the Sring password methods, use an enum flag instead. The bcrypt should happen before these methods are called. The DAO layer is *not* responsible for doing encryption.